napari/layers/_labels_layer/model.py

```diff
@@ -507,6 +501,19 @@ def get_message(self, coord, value):
 
         return msg
 
+    def _update_thumbnail(self):
+        """Update thumbnail with current image data and colors.
+        """
+        zoom_factor = np.divide(self._thumbnail_shape[:2],
+                                self._image_view.shape[:2]).min()
+        downsampled = np.round(ndi.zoom(self._image_view, zoom_factor,
+                                        prefilter=False, order=0))
+        downsampled = self.raw_to_displayed(downsampled)
+        mapped = self.colormap.map(downsampled) * 255
```

Comment:
I suggest removing the 255 here and then using `skimage.util.img_as_ubyte` to convert to uint8 and rescale in one call.

Reply:
done